products/fairphone.md

```diff
@@ -28,9 +28,9 @@ releases:
     link: https://shop.fairphone.com/fairphone-3
 
 -   releaseCycle: "2"
-    discontinued: 2023-03-01
-    eol: 2023-03-01
-    releaseDate: 2015-12-01
+    discontinued: 2023-03-08
```

Comment:
It was discontinued much earlier: https://www.fairphone.com/en/2019/03/22/proving-our-case-fairphone-2-is-sold-out/ (article is dated 22 March, but says 

> [A few are still available new from our partners](https://shop.fairphone.com/en/), while we have one remaining batch of refurbished units which will come back in stock in the coming weeks, but when those are sold, it will mark the end of Fairphone 2 sales.

so picking end of the month instead.

```suggestion
    discontinued: 2019-03-31
```